src/App.js

```diff
@@ -3,6 +3,7 @@ import {LandingPage} from "./pages/LandingPage/LandingPage";
 import {Route, Routes} from "react-router-dom";
 import {Login} from "../src/pages/Login/Login";
 import {Home} from "../src/pages/Home/Home";
+import {Archive} from "../src/pages/Archive/Archive";

```

Comment:
Use barrel import. Create index.js file in pages folder. If you don't know about it you can read here-
https://medium.com/@klauskpm/do-a-barrel-export-aa5b79b76b05